docs/config/validate-links.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ignorePatterns": [
+    { "pattern": "^https://mvnrepository\\.com" }
```

Comment:
I don't think we'll have links to mvnrepository.com in the JS SDK, so no need for this ignore pattern?